src/DesktopAnalytics/Analytics.cs

```diff
@@ -230,14 +230,31 @@ public static void Track(string eventName, Dictionary<string, string> properties
 		/// <param name="e"></param>
 		public static void ReportException(Exception e)
 		{
+			ReportException(e, null);
```

Comment:
Is this project using a late enough version of C# so we could just make the new parameter optional?